### DISCUSSION TITLE
why not add proxmox

### DISCUSSION THREAD

**Discussion by pippo73** [revenz/Fenrus#200] [Ideas]
Hello, 
I'm new to this system which I find interesting. I've noticed that you suppor several "apps" but not proxmox. Is it possibile to add it too?
tnx
SKAL

**Comment by revenz** [comment]
Its open source, so if someone wants to add support for it, theyre free to do so and submit a pull request

**Reply by pippo73** [reply]
I'm not a coder, but a poke-er . You can let this as a request for any good willing user :) or close it if you prefer

**Reply by revenz** [reply]
they meant adding proxmox as a smartapp to the dashboard so they could get info about proxmox, running VMs, CPU usage etc
